webapp-django/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -92,16 +96,25 @@ def check_crashids(entropy_chunk, date):
         page_iterator = paginator.paginate(Bucket=bucket, Prefix=raw_crash_key_prefix)
 
         for page in page_iterator:
-            for item in page.get("Contents", []):
-                raw_crash_key = item["Key"]
-                crash_id = raw_crash_key.split("/")[-1]
+            # NOTE(willkg): Keys look like /v2/raw_crash/ENTRPOY/DATE/CRASHID
+            crash_ids = [
+                item["Key"].split("/")[-1] for item in page.get("Contents", [])
+            ]
+
+            if not crash_ids:
+                continue
+
+            # Check S3 first
+            for crash_id in crash_ids:
                 if not is_in_s3(s3_client, bucket, crash_id):
                     missing.append(crash_id)
 
-                elif not is_in_elasticsearch(supersearch, crash_id):
-                    missing.append(crash_id)
+            # Check Elasticsearch in batches
+            for crash_ids_batch in chunked(crash_ids, 100):
+                missing_in_es = check_elasticsearch(supersearch, crash_ids_batch)
+                missing.extend(missing_in_es)
 
-    return missing
+    return list(set(missing))
```

Comment:
Because we're appending to `missing` for both the S3 and ES checks, it's possible for a crash id to be in there twice. So we turn it into a set to drop duplicates and back to a list to send back to the main process.